source/extensions/filters/http/transformation/transformation_filter.cc

```diff
@@ -205,6 +209,29 @@ void TransformationFilter::addEncoderData(Buffer::Instance &data) {
   encoder_callbacks_->addEncodedData(data, false);
 }
 
+void TransformationFilter::transformAccessLogs() {
```

Comment:
can this ever be used outside of access logging? name might be a bit too specific

Reply:
I think I agree, internally we should probably name this onStreamCompleteTransformation, and then we can decide what to call it in the user facing API